lab 2/css/style.css

```diff
@@ -0,0 +1,205 @@
+body {
+   margin: 0;
+   font-family: sans-serif;
+}
+
+a {
+   text-decoration: none;
+   color: white;
+}
+
+a:hover {
+   color: #c4c4c4;
+}
+
+header {
```

Comment:
I'd highly recommend you to use classes instead of tags in case of browser performance improvement and styles compatibility issues with other libs

Reply:
#done

Reply:
Could you please push your changes so I could see them here?

Reply:
Should i create a new pull request?
Or just push all to my master branch?

Reply:
well, I've figured out, that all commits, that are pushed to my master branch after the pull request, are automaticly updated here. So there is no need to answer my questions :)